README.md

```diff
@@ -230,7 +228,35 @@ export default {
 
 ## Reactivity
 
-vue-chartjs will update or re-render the chart if new data is passed.
+vue-chartjs will update or re-render the chart if new data or new options is passed.
```

Comment:
maybe just leave links to the website?

https://github.com/reactchartjs/react-chartjs-2#docs